README.md

```diff
@@ -180,9 +180,29 @@ To do this, `zgen` generates a static `init.zsh` file which does nothing but sou
 
 ### [Zinit](https://github.com/zdharma/zinit)
 
-**Zinit** is an innovative plugin manager with [semigraphical UI](https://github.com/zdharma/zplugin-crasis), [Turbo Mode](http://zdharma.org/zinit/wiki/INTRODUCTION/#turbo_mode_zsh_62_53), a clean fpath, reports, completion management and [services](https://github.com/zservices) support. You can use [crasis](https://github.com/zdharma/zplugin-crasis) or
-[declare-zsh](https://github.com/zdharma/declare-zsh) to edit your Zinit
-commands in your `~.zshrc` and also use [zinit-console](https://github.com/zinit-zsh/zinit-console) to view and change the state of the ZSH session and to delete the plugins and snippets from the disk.
+**Zinit** is an innovative and currently the
+[fastest](https://github.com/zdharma/pm-perf-test) plugin manager with support
```

Comment:
please don't embed line breaks inside paragraphs - I prefer to let the markdown renderer handle them, and it makes it easier when editing the file to move paragraphs around when they're single lines.

Reply:
I was editing in Vim and the long lines caused some problems, so I thought that I'll have them broke. It is withdrawn now!